docs/excel/custom-functions-json.md

```diff
@@ -21,6 +21,9 @@ However, you can make a custom functions project from scratch; it requires that
 
 This article will show you how to do all three of these steps.
 
+The following image explains the differences between using `yo office` scaffold files and writing JSON from scratch.
+![Image of differences between using Yo Office and writing your own JSON](../images/handauthoring.png)
+
 > [!NOTE]
 > In contrast with the `yo office` scaffold files, you need to hook your manifest up with the JSON file you create, through the `<Resources>` section in your XML manifest file. Note that the server settings on the server that hosts the JSON file must have [CORS](https://developer.mozilla.org/docs/Web/HTTP/CORS) enabled in order for custom functions to work correctly in Excel on the web.
```

Comment:
Unrelated nit: Try to avoid the word "hook", as it translates poorly. Consider "connect" or something similar.